meetings/2024-05-23/minutes.md

```diff
@@ -0,0 +1,195 @@
+
+This week's PQCA TAC included a presentation from Dana Wang, OpenSSF on open-source security, scorecards etc. In the future we hope we can influence these scorecards to capture if a project is pq ready/compliant/has CBOM.
+
+There's also activity to get funding for mentorship, and support students doing masters/PhD to make contributions. Doug mentioned this was part of the original formation discussions to bring new talent in.
+
+Finally there is currently a vote for vice-chair open.
+
+Nigel mentioned scorecard work had started in OQS, and whilst we have scorecards in pqcp, there is action to act on the findings.
+
+The potential of needing funding for formal audits was also brought up, but Matthias felt there is a fair way off yet. Ry pointed out that Trail Bits has recently joined PQCA. Doug mentioned OQS had a call with TrailBits recently, and hopefully they'll be looking at some oqs code pro bono.
+
+### Open Quantum Safe
+
+In addition to mentoring, audits (above) Doug pointed out that acting on scorecard findings takes engineering resources.
+
+Release 0.11 is being worked towards over the next month. Two likely inclusions will be stateful hash-based signatures, the Mayo signature scheme, and a formally verified Kyber implementation from libjade.
+
+A security issue was also reported which now has a patch.
+
+Ry pointed out that OQS has a [dashboard](https://openquantumsafe.org/dashboard) which can include links to OpenSSF reports. Nigel mentioned if we benchmark this could be useful info to add. Max pointed out we need to ensure the content is actionable, ie we use it.
+
+### What does assured mean
+
+We reviewed the discussion so far in [mlkem-c-aarch64#37](https://github.com/pq-code-package/mlkem-c-aarch64/issues/37)
+
+Minimum could be to run tests properly, use test vectors, run static analysis. If going for higher assurance we should do more - ie formal methods to prove no undefined behaviour, constant-time verification. Later extending to assembly correctness. Maybe we also look into additional tooling like CBMC especially when working on highly micro-optimized assembly.
+
+'production-ready'- very dependent on user's requirements ie embedded system vs cloud-scale service. Other aspects - like supply-chain security, performance, portability, BOMs are part of this too. Different implementations will vary (ie libjade etc is higher). Maybe alpha/pre-releases have lower guarantees. What's needed is to explain and document. can raise the bar over time.
+
+Manuel clarified a thought early on was to establish a taxonomy - or at least a set of terms that describe the levels of assurance.
+
+Rod pointed out that other industry assurance cases (safety critical etc) go with 'principles based assurance', or 'evidence based assurance' whereby you make a claim, and provide evidence to support - that may include tool used. Also clearly state assumptions. We should aim-high as seen with libjade, or (proprietary) AWS.
```

Comment:
The AWS work Rod was referring to is https://github.com/awslabs/s2n-bignum which is an open source (+ open proof) library of formally verified cryptographic primitives. 
```suggestion
Rod pointed out that other industry assurance cases (safety critical etc) go with 'principles based assurance', or 'evidence based assurance' whereby you make a claim, and provide evidence to support - that may include tool used. Also clearly state assumptions. We should aim-high as seen with libjade, or AWS' s2n-bignum.
```